cabal-install/Distribution/Client/SolverInstallPlan.hs

```diff
@@ -143,6 +144,9 @@ new indepGoals index =
 toList :: SolverInstallPlan -> [SolverPlanPackage]
 toList = Graph.toList . planIndex
 
+toMap :: SolverInstallPlan -> Map SolverId SolverPlanPackage
```

Comment:
Yeah, I thought I needed it, turned out I didn't, but it seemed generally useful so I didn't remove it.